r/R/dplyr-across.R

```diff
@@ -102,8 +102,20 @@ across_setup <- function(cols, fns, names, .caller_env, mask, inline = FALSE) {
     return(value)
   }
 
+  is_single_func <- function(fns) {
+
+    # function calls with package base::round
+    (is.call(fns) && fns[[1]] == as.name("::")) ||
```

Comment:
Do you need to check that `is.function(fns[[3]])` or something?

Reply:
I'm tempted to just allow R to catch it; if I try using something weird like a dataset instead of a function, we'll get something like `Warning: Expression ggplot2::diamonds(mpg) not supported in Arrow; pulling data into R`, and then the same error we'd get if trying this in a non-Arrow context.